e2e-tests/version-service/run

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+desc 'test when version service is reachable'
+cluster="version-service"
+create_namespace $namespace
+deploy_operator
+
+kubectl_bin apply -f $conf_dir/secrets.yml -f $conf_dir/client.yml
+
+IMAGE_MONGOD="percona/percona-server-mongodb:4.2.7-7"
+IMAGE_MONGOD_TO_UPDATE="percona/percona-server-mongodb:4.2.8-8"
```

Comment:
can we spin up Version service here? from some predictable tag with predictable the latest image?
reason - have a stably repeatable test

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/390/commits/a254bdffcaf3067fbbb642a3cdab1e82ed262272